src/schemas/section.ts

```diff
@@ -36,13 +38,13 @@ export const typeDefs = gql`
     "Indicates whether or not the section has changed since the template was last published"
     isDirty: Boolean!
     "The user who created the Object"
-    createdById: Int!
+    createdById: Int
     "The timestamp when the Object was created"
-    created: DateTimeISO!
+    created: String
```

Comment:
Changed created and modified to String, to fix an issue mariadb had with ISO date type.